### PR TITLE
Add an Autoplay example for amp-carousel autoplay, fixes #88

### LINKED ISSUE #88: amp-carousel autoplay sample

### PR DESCRIPTION
**Pull request by juliantoledo**
fixes #88

### CODE REVIEW

File: src/20_Components/amp-carousel.html [hunk 1]
```diff
@@ -40,6 +40,13 @@
     <amp-img src="/img/image3.jpg" width="400" height="300" layout="responsive"></amp-img>
   </amp-carousel>
 
+  <!-- Display a list of images as slides with autoplay. -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-carousel.html]: Can you mention which attributes are import.

**Comment by juliantoledo** [comment]
@sebastianbenz like that?

**Comment by sebastianbenz** [comment]
seems like the two PRs got mixed up.

**Comment by juliantoledo** [comment]
:( How did I do that?!

**Comment by sebastianbenz** [comment]
Did you submit them separately, but the new branch did contain the commit from the other PR?

**Comment by juliantoledo** [comment]
Has to be.

**Comment by sebastianbenz** [comment]
Seems like this one is outdated. Can you close it and only work on the other one?

**Pull request closed without merging**